tests/test_command_arf_to_graph.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pexpect
+import pytest
+from readchar import key
+
+import tests.any_test_help
+
+
+@pytest.mark.usefixtures("remove_generated_reports_in_root")
+def test_command_arf_to_graph():
+    out = os.popen(
+        ('python3 -m oval_graph.command_line arf-to-graph --off-web-browser ' +
```

Comment:
Don't use `os.system`, `os.spawn` or `popen` with string arguments. Use the subprocess module: https://docs.python.org/3/library/subprocess.html
The main reason for this is safety.